GNUmakefile

```diff
@@ -235,7 +235,7 @@ LDFLAGS_clean  = $(filter-out -fPIC, $(LDFLAGS))
 
 .PHONY: $(pkg)
 $(pkg):
-	perl -pe "s'#VERSION'2021.04.01'; \
+	perl -pe "s'#VERSION'2023.01.00'; \
```

Comment:
Before, it was April 1st, now it is January 0th. Is it okay?

Reply:
The version is slightly confusing. The last digits aren't day, but a counter. From the release script:

Tag is yyyy.mm.rr, where yyyy.mm is current year and month,
and rr is a release counter within current month, starting at 0.

So 2021.04.01 was the 2nd release in Apr 2021. There should be a 2021.04.00 release, too.